test/helpers/hosts_and_hostgroups_helper_test.rb

```diff
@@ -26,6 +29,8 @@ def test_accessible_lifecycle_environments
   end
 
   def test_accessible_lifecycle_environments_limited
+    @host.save
```

Comment:
Had to do this so I could call `@host.lifecycle_environment` later.. 
On the test environment with `FactoryBot.build` it doesn't work as the association needs to be saved in the database to work: https://github.com/Katello/katello/blob/master/app/models/katello/concerns/content_facet_host_extensions.rb#L29